### DISCUSSION TITLE
The Best Catch Phrase I've heard Today

### DISCUSSION THREAD

**Discussion by ghost** [EddieHubCommunity/support#124] [General]
Open source contributions a day keeps Eddie Away

**Comment by eddiejaoude** [comment]
thank you @KarenEfereyan 🎉 

I forgot about it already since the live stream only 3 hours again 🤦‍♂️ - I did say I need reminding. So thank you 💪

**Reply by eddiejaoude** [reply]
@KarenEfereyan your custom GitHub profile looks awesome 🔥 

I sent you an Invitation for our community GitHub Organisation

If you would like it public so it appears on your github profile for everyone else to see, after accepting you can do this by finding your name in the github organisation list and change the dropdown to public https://github.com/orgs/EddieJaoudeCommunity/people

**Reply by ghost** [reply]
Thanks Eddie. I'll still glean some more inspirations from you to make it even better.

**Comment by ghost** [comment]
Will do thanks for the awesome stream with Ruth. Loved Every bit of it

**Reply by eddiejaoude** [reply]
So good to hear 👍 . I am sure we will do a part 2 soon 🤓

**Comment by Vyvy-vi** [comment]
slight update to this: **OpenSource a day, keeps the Eddie away...**
(this has been used by Eddie on a few streams 😆 . I guess it's official)

**Reply by eddiejaoude** [reply]
LOL 😂 

My catch phrases are open source, so we can keep tweaking and improving them 🤓
